### PR TITLE
Redesign for Career page. 

### PR DESCRIPTION
**Pull request by codiejay**
Redesign for Career page. 
Previously we had just a text that said "Career page". But now we have implemented the design for career page WHEN there are no openings. 

Preview: https://monopedia-ui-git-312-design-careers-page-prediqt.vercel.app/static/careers
## Linked issues 

closes #312, closes #312, closes https://github.com/EveripediaNetwork/issues/issues/312

### CODE REVIEW

**Comment by Simplypheyie96** [comment]
@codiejay  I need the link where i can view the implementation. the visit preview link is taking back to the homepage

**Comment by Simplypheyie96** [comment]
> @codiejay I need the link where i can view the implementation. the visit preview link is taking back to the homepage

found it. never mind

**Review by Simplypheyie96 — APPROVED** [review]
Looks great!

**Comment by kesar** [comment]
I would suggest to remove the CTA it looks very redundant being so close to the other subscription button (same in figma)

image

image

cc: @Simplypheyie96 not big deal, but i guess in mobile looks very strange

**Comment by Simplypheyie96** [comment]
> I would suggest to remove the CTA it looks very redundant being so close to the other subscription button (same in figma)
> 
> image
> 
> image
> 
> cc: @Simplypheyie96 not big deal, but i guess in mobile looks very strange

oohh. okay. i will adjust that too

**Comment by kesar** [comment]
image
vs 
image

also title seems very off to the left compared w other pages

**Comment by Simplypheyie96** [comment]
> image vs image
> 
> also title seems very off to the left compared w other pages

The guidelines page was not designed in figma and was just brought from the old everipedia page. The careers title was designed in line with other titles on the everipedia page. like the activity page, the search result page etc.

**Comment by kesar** [comment]
image

image

image

the three headers are different, so I guess we need to get some kind of harmony w the titles

**Comment by Simplypheyie96** [comment]
@kesar should i open new issues for all of these?

File: src/components/Careers/NoOpeningsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      direction="column"
+      align="center"
+      justify="center"
+    >
+      <Image
+        flex={1}
+        marginInlineStart="0 !important"
+        src="/images/NoCarrersState.png"
+        w={{ base: '60%', sm: '50%', md: '55%', lg: '25%' }}
+      />
```
Comment (anubra266) [inline src/components/Careers/NoOpeningsCard.tsx:20]: We could skip this one, but it's far better and performant to use SVGs for our illustrations.

File: src/pages/static/careers.tsx [hunk 2]
```diff
@@ -1,5 +1,63 @@
+import NoOpenings from '@/components/Careers/NoOpeningsCard'
+import { Button, Flex, Heading, Text } from '@chakra-ui/react'
 import React from 'react'
+import { useTranslation } from 'react-i18next'
 
-const Careers = () => <main>Career page</main>
+const Careers = () => {
+  const { t } = useTranslation()
+  const [openings] = React.useState<Array<any> | null>()
+  console.log(openings)
```
Comment (anubra266) [inline src/pages/static/careers.tsx:9]: We should remove console logs

File: src/pages/static/careers.tsx [hunk 3]
```diff
@@ -1,5 +1,63 @@
+import NoOpenings from '@/components/Careers/NoOpeningsCard'
+import { Button, Flex, Heading, Text } from '@chakra-ui/react'
 import React from 'react'
+import { useTranslation } from 'react-i18next'
 
-const Careers = () => <main>Career page</main>
+const Careers = () => {
+  const { t } = useTranslation()
+  const [openings] = React.useState<Array<any> | null>()
```
Comment (anubra266) [inline src/pages/static/careers.tsx:8]: What's the purpose of this? It's always going to be undefined

**Review by anubra266 — CHANGES_REQUESTED** [review]

File: src/pages/static/careers.tsx (hunk 3 above)
Comment (codiejay) [reply src/pages/static/careers.tsx:8]: For now we don't really have live data to simulate with. So for now we have this and will refactor one we have a list of opening to loop through. (if i added a `[openings, setOpenings] `, I'll have a Type Error where I declared a var but never used it.

**Review by anubra266 — APPROVED** [review]

**Pull request merged**